lib/shoulda/matchers/active_model/validate_presence_of_matcher.rb

```diff
@@ -134,9 +134,10 @@ def simple_description
         private
 
         def secure_password_being_validated?
-          defined?(::ActiveModel::SecurePassword) &&
-            @subject.class.ancestors.include?(::ActiveModel::SecurePassword::InstanceMethodsOnActivation) &&
-            @attribute == :password
+          return false unless defined?(::ActiveModel::SecurePassword::InstanceMethodsOnActivation)
```

Comment:
This will cause `secure_password_being_validated?` to always return false on Rails 6 - but presumably this method exists for a reason, and it should continue to return true when the attribute being validated is a secure password.